apps/api/src/users/user.controller.ts

```diff
@@ -1,18 +1,23 @@
-import { Controller, Get, Query } from '@nestjs/common';
+import { Body, Controller, Get, Param, Post, Query, UseGuards } from '@nestjs/common';
 import { UsersService } from './users.service';
-// import { JwtAuthGuard } from '../auth/jwt-auth.guard';
-import { User } from 'types';
+import { JwtAuthGuard } from '../auth/jwt-auth.guard';
+import { Userfront } from 'types';
+import { isUUIDDto } from '../conversations/dtos/IsUUID.dto';
 
-//@UseGuards(JwtAuthGuard)
+@UseGuards(JwtAuthGuard)
 @Controller('user')
 export class UsersController {
   constructor(private readonly usersService: UsersService) {}
 
-  @Get()
-  async getFriendsList(
-    @Query() query: { user_id: string },
-  ): Promise<User | null> {
-    console.log(`user_id : ${query.user_id}`);
-    return this.usersService.getById(query.user_id);
+  @Get("/:id")
+  async getUser(@Param() { id } : isUUIDDto): Promise<Userfront | null> {
+    return this.usersService.getUser(id);
+  }
+
+  @Post('updateLevel')
```

Comment:
Making update level as post allow user to change their level without restriction...
This will need to change.

Reply:
en fait j'avais cette route juste pour tester l'update je l'ai delete.